src/Network/API/TSNetworkManager.m

```diff
@@ -59,7 +55,7 @@ - (void)makeRequest:(TSRequest *)request
     void (^failure)(NSURLSessionDataTask *task, NSError *error) =
         [TSNetworkManager errorPrettifyingForFailureBlock:failureBlock];
 
-    AFHTTPSessionManager *sessionManager = self.signalService.HTTPSessionManager;
```

Comment:
Because there is a dependency cycle.

`AppDelegate` creates `TSNetworkManager` which requires `OWSSignalService` which requires `TSAccountManager` which requires `TSNetworkManager`.  

What I'd like to do is lazy-initialize `OWSSignalService` but there isn't a good way to do that without duplicating/complicating much of its logic or running afoul of synchronization issues.

Happy to discuss.